AAI/AAIConnectProfile.md

```diff
@@ -298,8 +335,8 @@ the Broker.
         previously generated access tokens.
 
 6.  By signing an access token, an Broker asserts that the GA4GH Claims that
```

Comment:
If the passport-scoped access token is not a JWT, this paragraph may be removed.